README.md

```diff
@@ -49,10 +52,16 @@ git clone https://github.com/asterinas/asterinas
 2. Run a Docker container as the development environment.
 
 ```bash
-docker run -it --privileged --network=host --device=/dev/kvm -v ./asterinas:/root/asterinas asterinas/asterinas:0.3.1
+docker run -it --privileged --network=host --device=/dev/kvm -v ./asterinas:/root/asterinas asterinas/asterinas:0.4.0
+```
+
+3. Install the Asterinas OSDK from source.
+
+```bash
+make install_osdk
 ```
```

Comment:
Is it possible to hide this step? I really don't want to expose the concept of OSDK to a newcomer of Asterinas Kernel. He or she wants to build and test the kernel. That's all. He or she should not be forced to learn the new concept of an internal tool called OSDK that is used to build the kernel.

Reply:
We can have a explicit makefile dependency on `$CARGO_HOME/bin/cargo-osdk`.